src/betterproto/__init__.py

```diff
@@ -875,6 +874,10 @@ def parse(self: T, data: bytes) -> T:
                 )
 
             current = getattr(self, field_name)
+
+            if self.__raw_get(field_name) == PLACEHOLDER:
```

Comment:
This should use `is` not `==`

Reply:
And maybe there should be a return on this

Reply:
Why should there be a return here? The for loop has to continue

Reply:
Sorry I meant continue